docs/platforms/android/troubleshooting/index.mdx

```diff
@@ -4,6 +4,20 @@ description: "Troubleshoot and resolve common issues with the Android SDK."
 sidebar_order: 9000
 ---
 
+## Excluding unwanted resources from apk/aab
+
+The Sentry Android SDK bundles some resources transitively from the Sentry Java SDK that are not required for Android apps. It's not possible to exclude these resources when publishing the SDK, but you can add the following snippet to your `app/build.gradle` file to reduce the final apk size by a bit:
```

Comment:
`l`: is there a somewhat reasonable number/indication of how much this will shave off the apk in size? like "a few 100 KB"? might be useful for ppl to decide whether they want to try this

Reply:
38 bytes 😅 not sure if worth to mention haha